Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -36,6 +36,10 @@
     sprite: Clothing/OuterClothing/Coats/detective_grey.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Coats/detective_grey.rsi
+  - type: StorageFill
+    contents:
+    - id: SmokingPipeFilledTobacco
+    - id: FlippoEngravedLighter
```

Comment:
```suggestion
```
Doesn't it already inherit this from ClothingOuterCoatDetectiveLoadout?

Reply:
I believe it might but I'm honestly not experienced enough to know for certain, let me do a quick test

Reply:
Appears you are correct, removing that I still spawn in with the correct contents